internal/config/configuration.go

```diff
@@ -8,6 +8,14 @@ import (
 )
 
 type Configuration struct {
+
+	Server  ServerConfiguration
+	Mongodb MongodbConfiguration
+	Redis   RedisConfiguration
+	S3      S3StorageConfiguration
+	Python  MicroserviceConfiguration
+	Sentry  SentryConfiguration
+=======
 	Server      ServerConfiguration
 	Mongodb     MongodbConfiguration
 	Redis       RedisConfiguration
```

Comment:
why is ====== here? it's not meant to be in here at all, it's suppose to be a marker when resolving changes.
You also have multiple instances of Server, Mongodb and Redis.
This will definitely break the server